<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Махнатеев С.А. | Транслятор - Pascal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="btnOpenIR">
          <property name="text">
           <string>Выбрать файл</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnGenerate">
          <property name="text">
           <string>Сгенерировать</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <widget class="QPlainTextEdit" name="plainTextIR">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelSelectedIRFile">
        <property name="text">
         <string>Выбранный файл: Файл не выбран</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Внутреннее представление (Постфиксная запись)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="2" rowspan="3">
       <widget class="QPlainTextEdit" name="plainTextEditPascal"/>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Pascal</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelSelectedOriginalFile">
          <property name="text">
           <string>Выбранный файл: Файл не выбран</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="btnOpenOriginal">
          <property name="text">
           <string>Выбрать файл</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSaveOriginal">
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QPlainTextEdit" name="plainTextOriginal"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Оригинал</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btnSavePascal">
        <property name="text">
         <string>Сохранить сгенерированный</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btnDisassemble">
        <property name="text">
         <string>Разобрать</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
